Frontend/CO.CDP.OrganisationApp/Pages/OrganisationOverview.cshtml

```diff
@@ -35,9 +38,25 @@
                         </dt>
                         <dd class="govuk-summary-list__value">
                             <p class="govuk-body">
-                                @organisationDetails.Identifier.Scheme.SchemeDescription()
+                                @if (showPpon)
+                                {
+                                    @pponIndentifier!.Scheme.SchemeDescription()
+                                }
+                                else
+                                {
+                                    @organisationDetails.Identifier.Scheme.SchemeDescription()
+                                }
```

Comment:
Instead, of using `showPpon` can we define an `identifier` variable to show it here? This way we wouldn't need conditions in the template.

Reply:
Done.